src/Symfony/Component/Console/Tester/CommandTester.php

```diff
@@ -129,4 +137,39 @@ public function getStatusCode()
     {
         return $this->statusCode;
     }
+
+    /**
+     * Sets the user inputs.
+     *
+     * @param array A simple array of strings representing each input
+     *              passed to the command input stream.
+     *
+     * @return CommandTester
+     */
+    public function setInputs(array $inputs)
+    {
+        $this->inputs = $inputs;
+
+        return $this;
+    }
+
+    /**
+     * Creates the command input stream.
+     */
+    private function createInputStream()
```

Comment:
I would prefer to have it return the stream instead of injecting it via `setInputStream()`, which can be done in the caller.